spark/src/main/scala/org/apache/spark/sql/delta/util/Utils.scala

```diff
@@ -64,7 +66,7 @@ object Utils {
    * otherwise.
    */
   def try_element_at(mapColumn: Column, key: Any): Column = {
-    Column {
+    newColumn {
```

Comment:
Ok, I tried this out in 2.12 REPL and implicit constructors are possible:
```scala
implicit class ColumnConstructorExt(c: Column.type) {
  def apply(e: Expression): Column = ExpressionUtils.column(e)
}
```

Reply:
TIL :)... I was wondering if that is possible. Let me add that to OSS.

Reply:
Thanks @larsk-db. This simplifies the changes needed for this PR.